TockLibrary.mk

```diff
@@ -29,10 +29,6 @@ $(call check_defined, LIBNAME)
 $(call check_defined, $(LIBNAME)_DIR)
 $(call check_defined, $(LIBNAME)_SRCS)
 
-ifeq ($(strip $($(LIBNAME)_SRCS)),)
-  $(error Library "$(LIBNAME)" has no SRCS?)
-endif
-
```

Comment:
Do we need to remove this?

Reply:
Yep https://github.com/tock/libtock-c/pull/432#discussion_r1645043185